distribution/docs/src/main/resources/content/_metadataAttributes/associations-attributes-table.adoc

```diff
@@ -17,19 +17,19 @@
 |Constraints
 |Example Value
 
-|[[metacard.associations.derived]]metacard.associations.derived
+|[[_metacard.associations.derived]]metacard.associations.derived
```

Comment:
❓Why do we not use a prefix in the links that are wrapped by `[[` and `]]`?

Reply:
they are the destination. The ones wrapped `<<_xyz>>` are the links _to_ these anchors. The prefix will resolve to `document-name.adoc#_anchor`. It's used so we can publish a `managing.adoc` and a `reference.adoc` and they can have relative links between each other.